core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -1098,6 +1098,12 @@ public enum Property {
   @Experimental
   COMPACTOR_CLIENTPORT("compactor.port.client", "9133", PropertyType.PORT,
       "The port used for handling client connections on the compactor servers", "2.1.0"),
+  COMPACTOR_JOB_WAIT_TIME("compactor.wait.time.job", "1s", PropertyType.TIMEDURATION,
+      "The amount of time to wait between checks for the next compaction job", "4.0.0"),
```

Comment:
If we have a max the making this prop a min seems more consistent.

```suggestion
  COMPACTOR_JOB_WAIT_TIME_MIN("compactor.wait.time.job.min", "1s", PropertyType.TIMEDURATION,
      "The minimum amount of time to wait between checks for the next compaction job. Going up to a max of compactor.wait.time.job.max.", "4.0.0"),
```

Reply:
Renamed property and updated description in 66fc591.